cpp-rt.hpp

```diff
@@ -0,0 +1,97 @@
+#ifndef CPPRT_HPP
+#define CPPRT_HPP
+
+#include <stddef.h>
+#include <malloc.h>
+#include <cstdint>
+#include <cassert>
+
+#define FATAL(s) { fprintf(stderr, s); exit(1); }
+
+// This is a hack for .SO to work when they are compiled against a program/library that uses libstdc++
```

Comment:
Any chance to elaborate on this here? Probably adding links to resources would be nice too.

Reply:
Thanks!